src/olympia/users/utils.py

```diff
@@ -61,7 +61,10 @@ def get_task_user():
     """
     from olympia.users.models import UserProfile
 
-    return UserProfile.objects.get(pk=settings.TASK_USER_ID)
+    try:
+        return UserProfile.objects.get(pk=settings.TASK_USER_ID)
+    except UserProfile.DoesNotExist:
+        return None
```

Comment:
We don't want this change - if there's a test failing because the task user doesn't exist, just create a user profile with `TASK_USER_ID` as the id in the test. (use `user_factory`)